script/setup/critools-version

```diff
@@ -1 +1 @@
-v1.28.0
+master
```

Comment:
Is this temporary?

Reply:
yes, current cri-tools project having problme, i submit a pr to fix this: https://github.com/kubernetes-sigs/cri-tools/pull/1302, wait having a new release version, i can update to new version.